module/users/helper.js

```diff
@@ -1119,6 +1142,116 @@ module.exports = class UsersHelper {
       }
     });
   }
+    /**
+   * Get overall Stats of the User
+   * @method
+   * @name getAllStats
+   * @param {String} userToken - userToken
+   * @param {String} type - type
+   * @returns {Object} - returns overall stats or targeted records based on type
+   */
+  static getAllStats({ userToken, type }) {
+    
+    return new Promise(async (resolve, reject) => {
+      try {
+        if (type) {
+          switch (type) {
+            case "observation":
+              let observationInfo = await surveyService.getObservationInfo(userToken);
+
+              resolve({
+                type: type,
+                data: observationInfo.data,
+              });
+
+              break;
+            case "project":
+              let projectsStartedByUserAPICall =
+                improvementProjectService.listProjectOverviewInfo({
+                  stats: "false",
+                  userToken,
+                  userInvolvement:'creator'
```

Comment:
dont seggrigate like this @borkarsaish65

Reply:
done